src/main/java/org/folio/oaipmh/processors/MarcWithHoldingsRequestHelper.java

```diff
@@ -544,7 +544,7 @@ private void enrichDiscoverySuppressed(JsonObject itemsandholdingsfields, JsonOb
       ).forEach(instance -> {
       final String instanceId = instance.getString(INSTANCE_ID_FIELD_NAME);
```

Comment:
I realize this isn't something introduced in this PR but can you please fix the indentation here?